docs/content/rules/sort-classes.mdx

```diff
@@ -472,6 +473,34 @@ abstract class Example extends BaseExample {
 
 ### customGroups
 
+<Important title="ℹ️ Migrating from the old API">
```

Comment:
Let's remove the emoji from the documentation site after all, shall we? They're not used anywhere else.

Everything else looks good to me.

Reply:
@azat-io Fixed in b9d52b44ef5f298de55225a526e8566511db702b.